### PR TITLE
[DC-2147] Verify Personal Family Medical History joint survey questions and answers are removed

### PR DESCRIPTION
**Pull request by ksdkalluri**

### CODE REVIEW

File: data_steward/analytics/cdr_ops/rdr_export_qc.py [hunk 1]
```diff
@@ -642,4 +643,29 @@
...
+SELECT
+  observation_source_value,
+  COUNT(*) AS n_rows_violation
+FROM
+  `{{project_id}}.{{dataset}}.observation`
+WHERE
+  -- separated surveys were removed from participant portal on 2021-11-01 --
+  -- combined survey was launced on 2021-11-01 --
+  observation_date > '2021-10-31'
+  AND LOWER(observation_source_value) IN ({{combined_pfmhh_concepts}})
```
Comment (nishanthpp93) [inline data_steward/analytics/cdr_ops/rdr_export_qc.py:662]: Should this also check in `observation_source_concept_id`?

File: data_steward/analytics/cdr_ops/rdr_export_qc.py [hunk 2]
```diff
@@ -642,4 +643,29 @@
...
+
+tpl = JINJA_ENV.from_string("""
+SELECT
+  observation_source_value,
+  COUNT(*) AS n_rows_violation
+FROM
+  `{{project_id}}.{{dataset}}.observation`
+WHERE
+  -- separated surveys were removed from participant portal on 2021-11-01 --
+  -- combined survey was launced on 2021-11-01 --
```
Comment (nishanthpp93) [inline data_steward/analytics/cdr_ops/rdr_export_qc.py:660]: nit: typo `launched`

File: data_steward/analytics/cdr_ops/rdr_export_qc.py [hunk 3]
```diff
@@ -642,4 +643,29 @@
 group by value_source_concept_id, value_as_concept_id
 """)
 query = tpl.render(new_rdr=new_rdr, project_id=project_id)
-execute(client, query)
\ No newline at end of file
+execute(client, query)
+
+# # Verify that no joint Personal Family Medical History survey information exists in the Dataset.
```
Comment (nishanthpp93) [inline data_steward/analytics/cdr_ops/rdr_export_qc.py:648]: Can you add the ticket number here as well?

**Review by nishanthpp93 — COMMENTED** [review]
Overall looks good but added a couple comments.

File: data_steward/analytics/cdr_ops/rdr_export_qc.py (hunk 1 above)
Comment (ksdkalluri) [reply data_steward/analytics/cdr_ops/rdr_export_qc.py:662]: we only have a list of concept codes that we are using in the cleaning rule as well, and this is potentially a one-time thing. we are going to remove this step by nov CDR.

Comment (nishanthpp93) [reply data_steward/analytics/cdr_ops/rdr_export_qc.py:662]: Ah, okay. This is fine then.

**Review by nishanthpp93 — APPROVED** [review]
Looks good!

**Pull request merged**